content/post/kubedb-v2024.9.30/index.md

```diff
@@ -0,0 +1,979 @@
+
+Now apply a **Postgres** CR.
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: Postgres
+metadata:
+  name: ha-postgres
+  namespace: demo
+  labels:
+    archiver: "true"
+spec:
+  version: "16.1"
+  replicas: 3
+  clientAuthMode: "md5"
+  standbyMode: Hot
+  storageType: Durable
+  storage:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 3Gi
+  terminationPolicy: WipeOut
+```
+You should see your full-backup succeeded without the need of an external snapshotter. 
+
+Now you can use another **Postgres** CR for archive recovery.
+
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: Postgres
+metadata:
+  name: restore-pg
+  namespace: demo
+spec:
+  init:
+    archiver:
+      encryptionSecret:
+        name: encrypt-secret
+        namespace: demo
+      fullDBRepository:
+        name: ha-postgres-full
+        namespace: demo
+      manifestRepository:
+        name: ha-postgres-manifest
+        namespace: demo
+      recoveryTimestamp: "2024-09-25T06:36:13.982904Z" 
+  version: "16.1"
+  replicas: 3
+  standbyMode: Hot
+  storageType: Durable
+  storage:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 3Gi
+  terminationPolicy: WipeOut
+```
+
+
+Also we have added documentation for Postgresql **Arbiter node**, **Autoscaling** and for few Ops requests  **Restart**, **Reconfigure** and **Reconfigure TLS**.
+
+
+## SingleStore
+In this release, we have added support for manifest backups. Using Kubestash, you can able to manifest backup and restore for SingleStore.
+
+Here is the backupconfiguration yaml for manifest backup,
+
+```yaml
+apiVersion: core.kubestash.com/v1alpha1
+kind: BackupConfiguration
+metadata:
+  name: sample-singlestore-backup
+  namespace: demo
+spec:
+  target:
+    apiGroup: kubedb.com
+    kind: Singlestore
+    namespace: demo
+    name: sample-singlestore
+  backends:
+    - name: gcs-backend
+      storageRef:
+        namespace: demo
+        name: gcs-storage
+      retentionPolicy:
+        name: demo-retention
+        namespace: demo
+  sessions:
+    - name: frequent-backup
+      scheduler:
+        schedule: "*/5 * * * *"
+        jobTemplate:
+          backoffLimit: 1
+      repositories:
+        - name: gcs-singlestore-repo
+          backend: gcs-backend
+          directory: /singlestore
+          encryptionSecret:
+            name: encrypt-secret
+            namespace: demo
+      addon:
+        name: singlestore-addon
+        tasks:
+          - name: manifest-backup
+          - name: logical-backup
+```
+
+Here is  the restoresession yaml for restore manifest-restore,
+
+```yaml
+apiVersion: core.kubestash.com/v1alpha1
+kind: RestoreSession
+metadata:
+  name: restore-sample-singlestore
+  namespace: demo
+spec:
+  manifestOptions:
+    restoreNamespace: dev
```

Comment:
`restoreNamespace` should be the sub-field of `singlestore`.
```
singlestore:
  restoreNamespace: dev
```

Also do not need to specify `db: true`. Its default value is true.